common/include/pcl/common/common_headers.h

```diff
@@ -35,6 +35,7 @@
 
 #pragma once
 
+PCL_DEPRECATED_HEADER(1, 15, "Please include any needed headers from pcl/common directly.")
```

Comment:
We can let others include the common headers, no issues. OpenCV also provides similar functionality of per-module headers

Reply:
I would always recommend to only include the headers that are actually needed, but I removed the deprecation notice again.